src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp

```diff
@@ -522,6 +530,10 @@ namespace OpenMS
 
     if (!svm_probs_internal_.empty()) calculateFDR_(features);
 
+    //TODO MRMFeatureFinderScoring already does an ElutionModel scoring. It uses EMG fitting.
```

Comment:
@hendrikweisser

Reply:
Yeah but this is just a small optimization that we could do. So not too urgent. I think the scoring FFID is more elaborate also.

Reply:
The elution model in FFId is used for quantification and filtering. In MRMFeatureFinderScoring, is the model also fit to multiple mass traces at once? And is there a way to pass on the model parameters (or are they lost after the score is calculated)?

Reply:
No, exactly, the FFID model does simultaneous optimization, while the other doesn't. And yes, the model parameters are currently gone. My favorite solution would be to use the FFID model during Swath scoring already and save the parameters. What do you need the params for?